### ISSUE TITLE
cannot find

### ISSUE THREAD

**Issue by zouxiaomingya** [react-monaco-editor/react-monaco-editor#226]
Cannot find module 'monaco-editor/esm/vs/editor/standalone/browser/accessibilityHelp/accessibilityHelp'
 ------
Webpack configuration error

**Comment by domoritz** [comment]
Please provide a complete example to reproduce the error.

**Comment by DanceParty** [comment]
Upgrading from v0.20.0 -> latest caused this error for me. Not sure why. Also getting error when importing `monaco-editor` that it cannot find that module.
